kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration/src/main/java/org/kie/server/springboot/autoconfiguration/KieServerAutoConfiguration.java

```diff
@@ -76,7 +80,8 @@ public class KieServerAutoConfiguration extends AbstractJaxrsClassesScanServer {
     private String cxfPath;
 
     private static final Logger logger = LoggerFactory.getLogger(KieServerAutoConfiguration.class);
-    
+    private String EXPLODED_JAR_FOLDER = System.getProperty("org.kie.maven.resolver.folder");
```

Comment:
Probably it's better to use value annotation:
```suggestion
    @Value("${org.kie.maven.resolver.folder:/}")
    private String EXPLODED_JAR_FOLDER;
```